src/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import FHIR from 'fhirclient';
+
+const root = ReactDOM.createRoot(
+  document.getElementById('root') as HTMLElement
+);
+
+interface App {
+  client?: React.ReactNode;
+}
+
+const smartLaunch = () => {
+
+  FHIR.oauth2
+    .init({
```

Comment:
using init is fine for now but we should all just be aware of the specific details around using it. http://docs.smarthealthit.org/client-js/api#initoptions-promiseclient-experimental

Reply:
follow up task in backlog